src/kernels/types.ts

```diff
@@ -579,7 +581,7 @@ export interface IJupyterKernelSession extends IBaseKernelSession<'remoteJupyter
 export interface IRawKernelSession extends IBaseKernelSession<'localRaw'> {}
 export type IKernelSession = IJupyterKernelSession | IRawKernelSession;
 
-export type ISessionWithSocket = Session.ISessionConnection & {
+export interface ISessionWithSocket extends Session.ISessionConnection {
```

Comment:
Changed type to interface so its easier to identify usages of individual properties (i.e. improvement for finding references in VSCode, via code lenses)